### PR TITLE
Removed excludes from build.gradle for message files

### PR DESCRIPTION
**Pull request by ajaypaul-ibm**
Removed excludes from build.gradle for message files and made changes in resources as required.
for #27500

### CODE REVIEW

**Comment by ajaypaul-ibm** [comment]
#libby
#build

File: dev/io.openliberty.persistence.3.1.thirdparty/resources/org/eclipse/persistence/exceptions/i18n/ValidationExceptionResource.nlsprops [hunk 1]
```diff
@@ -633,7 +633,7 @@
 
 7356=Procedure: [{1}] cannot be executed because {0} does not currently support multiple out parameters
 
-7357=The [{0}] URL for the [{1}] resource does not belong to a valid persistence root, as defined by the JPA Specification.
+7357=The "[{0}]" URL for "[{1}]" resource does not belong to a valid persistence root (as per Jakarta Persistence Specification)
```
Comment (helyarp) [inline dev/io.openliberty.persistence.3.1.thirdparty/resources/org/eclipse/persistence/exceptions/i18n/ValidationExceptionResource.nlsprops:636]: To remove the parens and "as per" and reintroduce "the" before the resource variable and spec name, pls change the message to:

The "[{0}]" URL for the "[{1}]" resource does not belong to a valid persistence root, as defined by the Jakarta Persistence specification

Add a period if one is needed at the end of the message.

File: dev/io.openliberty.persistence.3.1.thirdparty/resources/org/eclipse/persistence/internal/localization/i18n/LoggingLocalizationResource.nlsprops [hunk 2]
```diff
@@ -727,7 +727,7 @@ validate_object_space=validate object space.
 
 verifiy_columns_read_locking_not_supported=Optimistic setting "Read" on "verify-columns" in entity ({0}) is not directly supported in EclipseLink CMP. See migration doc for details.
 
-verifiy_rows_read_locking_not_supported=Optimistic setting "Read" on "verify-rows" in entity ({0}) is not directly supported in EclipseLink CMP. See migration doc for details.
+verifiy_rows_read_locking_not_supported=Optimistic setting "Read" on "verify-rows" in entity ({0}) is not directly supported in EclipseLink CMP. See migration doc for details..
```
Comment (helyarp) [inline dev/io.openliberty.persistence.3.1.thirdparty/resources/org/eclipse/persistence/internal/localization/i18n/LoggingLocalizationResource.nlsprops:730]: Remove extra period

**Review by helyarp — COMMENTED** [review]
Suggested changes

**Comment by tkburroughs** [comment]
It appears the message improvements in open-liberty have not been contributed to eclipselink yet; so I'd recommend cancelling this PR, contributing the message improvements to eclipselink 4.0 and master branches and then remove the excludes from  `build.gradle` with the update to 4.0.3.

**Comment by tkburroughs** [comment]
See this eclipselink issue for the 4.0 contribution: https://github.com/eclipse-ee4j/eclipselink/pull/2055

**Comment by ajaypaul-ibm** [comment]
Closing this PR as this will not be completed until the message changes are included in a new EclipseLink version.

**Pull request closed without merging**